.vscode/settings.json

```diff
@@ -5,5 +5,6 @@
 	"editor.insertSpaces": false,
 	"editor.detectIndentation": false,
 	"files.eol": "\n",
-	"typescript.tsdk": "node_modules\\typescript\\lib"
+	"typescript.tsdk": "node_modules\\typescript\\lib",
+	"testing.automaticallyOpenPeekView": "never"
```

Comment:
What is this setting?

Reply:
Uhhhhhhhhhhhhhhhhhhhhhhhhhhhh

Something the Vitest extension added when I tried to run just the Result tests